### ISSUE TITLE
"TV Turned on!" logging

### ISSUE THREAD

**Issue by kbardai** [merdok/homebridge-webos-tv#310]
The logging for "TV Turned on!" seems to log every 2 minutes, don't know if this is intentional.

```
10/8/2020, 7:02:08 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 7:04:08 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 7:06:08 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 7:08:03 PM [webostv] [Master Bedroom TV] Pixel refresher started!
10/8/2020, 7:14:03 PM [webostv] [Master Bedroom TV] TV turned off!
10/8/2020, 7:14:19 PM [webostv] [Master Bedroom TV] Disconnected from TV
10/8/2020, 7:14:24 PM [webostv] [Master Bedroom TV] Starting TV alive polling
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] Connected to TV
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] Got TV information
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] TV webOS version: 3.0
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] App launched, current appId: com.webos.app.hdmi1
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] Sound output changed. Current sound output: tv_speaker
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] Subscribed to TV services
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] Connected to remote pointer input socket
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] Setup finished
10/8/2020, 8:38:26 PM [webostv] [Master Bedroom TV] TV setup finished, ready to control tv
10/8/2020, 8:40:21 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:42:21 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:45:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:47:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:49:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:51:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:53:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:55:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:57:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 8:59:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:01:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:03:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:05:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:07:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:09:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:11:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:13:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:15:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:17:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:19:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:21:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:23:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:25:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:27:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:29:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:31:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:33:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:35:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:37:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:39:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:41:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:43:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:45:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:47:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:49:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:51:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:53:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:55:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:57:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 9:59:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 10:01:16 PM [webostv] [Master Bedroom TV] TV turned on!
10/8/2020, 10:01:53 PM [webostv] [Master Bedroom TV] Pixel refresher started!
10/8/2020, 10:02:50 PM [webostv] [Master Bedroom TV] TV turned off!
10/8/2020, 10:02:52 PM [webostv] [Master Bedroom TV] Disconnected from TV
10/8/2020, 10:02:57 PM [webostv] [Master Bedroom TV] Starting TV alive polling
```

**Comment by merdok** [comment]
Hmmm, should not be like that. Can you enable homebridge debug  and also set the deepDebugLog to true and post the result here?

**Comment by rwgunther** [comment]
I am seeing the same behavior and was curious if it was supposed to work that way. I’m using the power state as a trigger, so I was concerned it might conflict with any subsequent actions on accessories triggered by that state, but it doesn’t appear to re-trigger anything, so I figured it’s just ongoing status monitoring.

**Comment by merdok** [comment]
This should only be a log thing and should not affect anything else, anyway i will be looking into that soon.

**Comment by merdok** [comment]
Should be fixed in Version 2.0.8
